Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04 as base
```

Comment:
Can you please separate this Dockerfile into multiple stages (build and package)? You also do not necessarily need to set the CMD or ENTRYPOINT macros.